metadata.json

```diff
@@ -103,7 +103,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 4.5.2 < 5.0.0"
```

Comment:
PE 2016.4 (LTS) includes 4.7 but the previous release 2016.2 (they skipped .3) includes 4.5.2... but according to https://docs.puppet.com/pe/latest/overview_getting_support.html 2016.2 is EOL April 30, 2017 so we could probably just jump straight to 4.7.0 support-wise, and use the older puppet-3 versions of modules for supporting 2016.2 and earlier until they EOL.